nostr-java-test/src/test/java/nostr/test/event/ApiEventTest.java

```diff
@@ -280,7 +280,7 @@ void testNIP57CreateZapRequestEvent() throws NostrException {
         final String ZAP_REQUEST_CONTENT = "zap request content";
         final Long AMOUNT = 1232456L;
         final String LNURL = "lnUrl";
-        final String RELAYS_TAG = "relaystag";
+        final String RELAYS_TAG = "ws://localhost:5555";
```

Comment:
Does this assume that a local relay must run on port 5555 for the unit test to work? 

cc: @avlo , @kuiperanon

Reply:
no- it's exclusively/simply a test of:
1) proper ZapRequestEvent object creation (via api.NIP57<>.createZapRequestEvent())
2) it's requisite fields populated, and
3) field values validated